### PR TITLE
Code improvements

### PR DESCRIPTION
**Pull request by Jay-o-Way**
As suggested in PR 121.
Zero visual or functional changes.

- remove unused namespaces
- use `wpfui` everywhere
- apply Intellisense suggestions
- loose changes
- rename methods

### CODE REVIEW

**Comment by U-C-S** [comment]
Thank you, looks good...

Will try it out locally once today or soon, Want to test couple of things.

File: Source/Hurl.BrowserSelector/Windows/MainWindow.xaml.cs [hunk 1]
```diff
@@ -127,33 +127,33 @@ private void LinkCopyBtnClick(object sender, RoutedEventArgs e)
...
-        this.Hide();
+        WindowState = WindowState.Minimized;
+        Hide();
     }
 
-    public void MaximizeWindow()
+    public void ShowWindow()
     {
-        this.Show();
         PositionWindowUnderTheMouse();
```
Comment (U-C-S) [inline Source/Hurl.BrowserSelector/Windows/MainWindow.xaml.cs:141]: Keep `Show` call on the top..... else, the window will only show up in background

File: Source/Hurl.BrowserSelector/Windows/MainWindow.xaml.cs [hunk 2]
```diff
@@ -127,33 +127,33 @@ private void LinkCopyBtnClick(object sender, RoutedEventArgs e)
...
-        this.WindowState = WindowState.Normal;
+        WindowState = WindowState.Normal;
+        Show();
```
Comment (U-C-S) [inline Source/Hurl.BrowserSelector/Windows/MainWindow.xaml.cs:143]: Keep `Show` call on the top..... else, the window will only show up in background

Comment (Jay-o-Way) [reply Source/Hurl.BrowserSelector/Windows/MainWindow.xaml.cs]: Done

**Pull request merged**